tests/post/steps/test_volume.py

```diff
@@ -179,6 +195,31 @@ def _check_pv():
         name='checking the absence of PersistentVolume {}'.format(name)
     )
 
+
+@then(parsers.parse("the PersistentVolume '{name}' is marked for deletion"))
+def check_pv_deletion_timestamp(name, k8s_client):
+    def _check_deletion_timestamp():
+        try:
+            pv = k8s_client.read_persistent_volume(name)
```

Comment:
Like you have a `_get_volume` helper, you should have a `_get_pv` helper.

```python
def _get_volume(k8s_client, name):
    try:
        return k8s_client.read_persistent_volume(name)
    except (ApiException, HTTPError) as exc:
        if isinstance(exc, ApiException) and exc.status == 404:
            return None
        raise
```

then you could write:
```python
    def _check_deletion_timestamp():
        pv = _get_pv(name)
        assert ps is not None, 'PersistentVolume {} not found'.format(name)
        assert pv.metadata.deletion_timestamp is not None,\
            'PersistentVolume {} is not marked for deletion'.format(name)
```